app/utils/discogs.schemas.ts

```diff
@@ -1,16 +1,29 @@
 import z from "zod";
 
-const AlbumSchema = z.object({
+const DiscogsAlbumSchema = z.object({
   id: z.number(),
   cover_image: z.string(),
   title: z.string(),
   year: z.string().optional(),
 });
 
-export type Album = z.infer<typeof AlbumSchema>;
+const SearchAlbumSchema = z.object({
+  id: z.number(),
+  cover_image: z.string(),
```

Comment:
good idea to convert everything to camelCase for your schema

```suggestion
  coverImage: z.string(),
```